src/modules/history/HistoryWrapper.tsx

```diff
@@ -154,6 +203,48 @@ export const HistoryWrapper = () => {
   const isEmpty = filteredTxns.length === 0;
   const filterActive = searchQuery !== '' || filterQuery.length > 0;
 
+  const transactionDownloadButton = (
+    text: string,
+    fileDownloadState: downloadState,
+    downloadHandler: () => void
+  ) => {
+    return (
+      <CustomTooltip
+        title={fileDownloadState === downloadState.ERROR ? downloadError : ''}
+        arrow={true}
+        wrap={true}
+        placement="top"
+      >
+        <Button
+          // Arul: To be uncommented
+          // disabled={isEmpty}
+          onClick={downloadHandler}
+        >
+          {fileDownloadState === downloadState.NEUTRAL ? (
+            <SvgIcon>
+              <DocumentDownloadIcon width={22} height={22} />
+            </SvgIcon>
+          ) : fileDownloadState === downloadState.LOADING ? (
+            <CircularProgress size={16} sx={{ mr: 2 }} color="inherit" />
+          ) : fileDownloadState === downloadState.ERROR ? (
+            <SvgIcon>
+              <WarningRounded width={22} height={22} />
+            </SvgIcon>
+          ) : (
+            fileDownloadState === downloadState.SUCCESS && (
+              <SvgIcon>
+                <CheckIcon width={22} height={22} />
+              </SvgIcon>
+            )
+          )}
+          <Typography variant="buttonM">
+            <Trans>{text}</Trans>
```

Comment:
The translations won't be applied by passing an object, to fix this you can change the type of `text` to `ReactNode` and wrap the parameter in `<Trans></Trans>` tag when it's passed

Reply:
I've updated the code in this [commit](https://github.com/aave/interface/pull/1680/commits/46e90c5a6abc13ebaf115bdc15ed084b8e4cfbbc#diff-570340e9ee2f9119e2473df26afed9054e35f28f15b6a4a249aed03fd4b965b9)